README.md

```diff
@@ -76,9 +76,9 @@ To help inform our roadmap, keep adopters apprised of major updates and changes
 
 ## Known Caveats/Issues
 
-__Remote Request Timeouts__ - With external connections, HTTP requests are sent back and forth - creating posts, transfering images, syncing post updates, etc. In certain situations, mostly commonly when distributing posts with a large number of images (or very large file sizes), using poorly configured or saturated servers / hosts, or using plugins that add significant weight to post creation, Distributor requests can fail. Although we do some error handling, there are certain cases in which post distribution can fail silently. If distribution requests are taking a long time to load and/or failing, consider adjusting the timeout; you can filter the timeout for pushing external posts [here](https://github.com/10up/distributor/blob/master/includes/classes/ExternalConnections/WordPressExternalConnection.php#L487). More advanced handling of large content requests, and improved error handling is on the road map for a future update.
+__Remote Request Timeouts__ - With external connections, HTTP requests are sent back and forth - creating posts, transfering images, syncing post updates, etc. In certain situations, mostly commonly when distributing posts with a large number of images (or very large file sizes), using poorly configured or saturated servers / hosts, or using plugins that add significant weight to post creation, Distributor requests can fail. Although we do some error handling, there are certain cases in which post distribution can fail silently. If distribution requests are taking a long time to load and/or failing, consider adjusting the timeout; you can filter the timeout for pushing external posts using the `dt_push_post_timeout` filter [here](https://github.com/10up/distributor/blob/master/includes/classes/ExternalConnections/WordPressExternalConnection.php#L620). More advanced handling of large content requests, and improved error handling is on the road map for a future update.
```

Comment:
Alternatively, we can use links with commit hash, the content of those lines will be always relevant. For example: `https://github.com/10up/distributor/blob/f7b60740e679bce4671ccd69a670abadce4f2f93/includes/classes/ExternalConnections/WordPressExternalConnection.php#L620`

Reply:
Only issue with the commit hash is that the remaining code may not be the current released code.  Maybe we link to https://10up.github.io/distributor/dt_push_post_timeout.html instead?

Reply:
If we do so, we need to keep the docs up to date as well. The current filter you linked above is out of date 😅: https://10up.github.io/distributor/includes_classes_ExternalConnections_WordPressExternalConnection.php.html#line605

Reply:
Do we have any task to regenerate the docs on release?

Reply:
@dinhtungdu looks like they get regenerated on push to `develop`: https://github.com/10up/distributor/blob/develop/.github/workflows/build-docs.yml

Reply:
Build docs actions are failing: https://github.com/10up/distributor/actions <- That's why it's out of date on the docs. But I agree that the best solution here is linking to the docs site. We just need to fix that action.

Reply:
Note that I'm looking into refreshing the GitHub secret used by that GitHub Action to see if that helps resolve the Build Hook Docs action's errors.  Assuming that works, I think we should link into those docs for this filter.

Reply:
I went ahead and changed the links to the docs site, we'll work to resolve the failing GitHub Action separately.